app/views/team_mailer/user_added_to_team.html.erb

```diff
@@ -1,5 +1,5 @@
 <h1><%= t('mailer.hello', firstname: @user.first_name) %></h1>
-<p><%= t('mailer.team_mailer.user_added_to_team.added_to_team_info_html',
+<p><%= t('.added_to_team_info_html',
          team_link: link_to(@team.name, team_url(@team))) %></p>
-<p><%= t('mailer.team_mailer.user_added_to_team.have_fun') %></p>
+<p><%= t('mailer.have_fun') %></p>
```

Comment:
@conradlempert @tomkellog See comment to `team_mailer/team_registered_to_event.html.erb`